warp10/src/main/java/io/warp10/script/binary/BitwiseAND.java

```diff
@@ -37,8 +40,44 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
     
     if (op2 instanceof Long && op1 instanceof Long) {
       stack.push(((Long) op1).longValue() & ((Long) op2).longValue());
+    } else if (op1 instanceof GeoTimeSerie && op2 instanceof GeoTimeSerie) {
+      GeoTimeSerie gts1 = (GeoTimeSerie) op1;
+      GeoTimeSerie gts2 = (GeoTimeSerie) op2;
+      if (GeoTimeSerie.TYPE.LONG == gts1.getType() && GeoTimeSerie.TYPE.LONG == gts2.getType()) {
+        GeoTimeSerie result = new GeoTimeSerie(Math.max(GTSHelper.nvalues(gts1), GTSHelper.nvalues(gts2)));
+        result.setType(GeoTimeSerie.TYPE.LONG);
+        GTSOpsHelper.GTSBinaryOp op = new GTSOpsHelper.GTSBinaryOp() {
```

Comment:
It is better if you allocate this in the class once and for all instead of creating a new instance each time apply is called.